docs/data/base/components/number-field/number-field.md

```diff
@@ -190,46 +189,40 @@ The pointer is locked while scrubbing, allowing the user to scrub infinitely wit
 </NumberField.ScrubArea>
 ```
 
-In your CSS, ensure any `label` elements inside the `ScrubArea` specify `cursor: unset`. You can rotate the above macOS-style cursor 90 degrees using a `transform` style.
+In your CSS, ensure any `<label/>` elements inside the `ScrubArea` specify `cursor: unset`. You can rotate the above macOS-style cursor 90 degrees using a `transform` style.
 
 :::info
 In Safari, the pointer is not locked. However, this doesn't affect the ability to scrub infinitely.
 :::
 
-#### Teleport distance
+### Teleport distance
 
-To teleport the virtual cursor closer to the input rather than the entire viewport, use the `teleportDistance` prop:
+Rather than teleporting the virtual cursor at the viewport boundary, you can use the `teleportDistance` prop to teleport the cursor at a custom boundary.
 
 ```js
 <NumberField.ScrubArea teleportDistance={200}>
   <NumberField.ScrubAreaCursor />
 </NumberField.ScrubArea>
 ```
 
-This specifies in pixels the distance the cursor can travel around the center of the scrub area element before it loops back around.
+This specifies the `px` distance the cursor can travel from the center of the scrub area element before it loops back around.
 
-#### Wheel scrubbing
+### Wheel scrubbing
 
-To allow the input to be scrubbed using the mouse wheel, add the `allowWheelScrub` prop:
+To allow the input to be scrubbed using the mouse wheel, add the `allowWheelScrub` prop.
 
 {{"demo": "UnstyledNumberFieldWheelScrub.js"}}
 
-## Hook
-
-```js
-import { useNumberField } from '@base_ui/react/useNumberField';
-```
+## Overriding default components
 
-The `useNumberField` hook lets you apply the functionality of a Number Field to a fully custom component.
-It returns props to be placed on the custom component, along with fields representing the component's internal state.
+Use the `render` prop to override the rendered elements with your own components.
 
-:::info
-Hooks give you the most room for customization, but require more work to implement.
-With hooks, you can take full control over how your component is rendered, and define all the custom props and CSS classes you need.
+```jsx
+<NumberField.Input render={(props) => <MyCustomInput {...props} />}> />
+```
 
-You may not need to use hooks unless you find that you're limited by the customization options of their component counterparts—for instance, if your component requires significantly different [HTML structure](#anatomy).
-:::
+All subcomponents accept the `render` prop.
 
 ## Accessibility
 
-Ensure the number field has an accessible name via a `label` element.
+Ensure the number field has an accessible name via a `<label/>` element.
```

Comment:
I've already changed this based on conflicting feedback, but will go with this suggested change since this is what I had initially. We can get our ducks in a row later on I guess.